geode-core/src/integrationTest/java/org/apache/geode/distributed/internal/membership/gms/fd/GMSHealthMonitorJUnitTest.java

```diff
@@ -191,17 +191,13 @@ public void tearDown() {
   }
 
   @Test
-  public void testHMServiceStarted() throws IOException {
+  public void testHMServiceStarted() throws IOException, Exception {
```

Comment:
There's no `IOException` thrown in this method (and some others).  Since these are all tests, I would suggest just using just `Exception`

Reply:
found a couple others and removed them as well...